dns_check/manifest.json

```diff
@@ -2,6 +2,7 @@
   "maintainer": "help@datadoghq.com",
   "manifest_version": "1.0.0",
   "name": "dns_check",
+  "display_name": "DNS Check",
```

Comment:
We should remove `Check`, we use it in a few integrations, but I don't think it makes sense.